content/docs/user-guide/gems/reference/ros2/_index.md

```diff
@@ -0,0 +1,119 @@
+---
+linkTitle: ROS2
+title: ROS2 Gem
+description: The ROS 2 Gem helps to build robotic simulations with ROS 2 / Robot Operating System.
+toc: true
+---
+
+<!-- # O3DE ROS2 Gem -->
+
+This Gem enables users to develop robotic simulations through ROS2 tools and components.
```

Comment:
"ROS2" vs "ROS 2" consistency. 

The name of the gem is ROS2 since a space in the name is not a great idea, but otherwise we should use "ROS 2"